bin/releasenotes-orig-urls.txt

```diff
@@ -0,0 +1,14 @@
+# Add the plugin release notes github url here
+https://github.com/opendistro-for-elasticsearch/anomaly-detection/blob/master/release-notes/opendistro-for-elasticsearch-anomaly-detection.release-notes-1.10.0.0.md
```

Comment:
Why are we using branches again instead of tags?
eg: https://github.com/opendistro-for-elasticsearch/anomaly-detection/blob/v1.10.1.0/release-notes/opendistro-for-elasticsearch-anomaly-detection.release-notes-1.10.0.0.md

Reply:
Also it is possible to not hardcode this? Additional work to change it every release! Maybe use **cut-version** and use it here

Reply:
1. All these links are retrieved from the checklist, if devs put the checklist link, that is why I use it here.
2. It is definitely not additional work, it is just a copy paste from checklist using less then 1 second. In fact, I will even remove all the links and only leave the comments there. This is a locally run script, not requiring anything else online to work.